File: read_excel.js
const xlsx = require('xlsx');

try {
    const workbook = xlsx.readFile('Formula F1NTA 2026.xlsx');
    console.log("Fogli presenti:", workbook.SheetNames);
    
    workbook.SheetNames.forEach(sheetName => {
        console.log(`\n\n=== FOGLIO: ${sheetName} ===`);
        const sheet = workbook.Sheets[sheetName];
        const data = xlsx.utils.sheet_to_json(sheet, { header: 1 });
        
        // Stampiamo le prime 20 righe del foglio
        for(let i=0; i<Math.min(20, data.length); i++) {
            console.log(JSON.stringify(data[i]));
        }
    });

} catch (error) {
    console.error("Errore durante la lettura:", error);
}

File: read_excel.py
import pandas as pd
import sys

try:
    file_path = "Formula F1NTA 2026.xlsx"
    xls = pd.ExcelFile(file_path)
    print("Sheets available:", xls.sheet_names)
    for sheet in xls.sheet_names:
        print(f"\n--- Sheet: {sheet} ---")
        df = pd.read_excel(file_path, sheet_name=sheet, nrows=20)
        print(df.head(10).to_string())
except Exception as e:
    print(f"Error: {e}")
